### ISSUE TITLE
Site usability on mobile

### ISSUE THREAD

**Issue by lucas-langa** [AdobeDocs/commerce-php#65]
The site content is not properly viewable on mobile. I have to scroll left and right to read a sentence

**Comment by hguthrie** [comment]
@lucas-langa  Are you still experiencing this issue? Our team has made some recent changes, so just checking if this is still an issue.

@bdenham  Can you verify any viewing issues?

**Comment by lucas-langa** [comment]
@hguthrie I will check

**Comment by bdenham** [comment]
Yes, there are viewing issues on smaller screen sizes, especially mobile devices.

**Comment by jeff-matthews** [comment]
This is an issue in the theme that we use, which is stored in another repo: https://github.com/adobe/aio-theme. I recommend reporting the issue there.
